gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import abc
+import re
+
+from apache_beam.io.gcp.internal.clients import bigquery
+from apitools.base.py import exceptions
+from oauth2client.client import GoogleCredentials
+
+
+__all__ = ['VariantTransformOptions', 'VcfReadOptions', 'BigQueryWriteOptions',
+           'FilterOptions', 'MergeOptions']
+
+_NONE_STRING = 'NONE'
+_MOVE_TO_CALLS_STRING = 'MOVE_TO_CALLS'
+_MERGE_WITH_NON_VARIANTS_STRING = 'MERGE_WITH_NON_VARIANTS'
+# List of supported merge strategies for variants.
+# - NONE: Variants will not be merged across files.
+# - MOVE_TO_CALLS: uses libs.variant_merge.move_to_calls_strategy
+#   for merging. Please see the documentation in that file for details.
+_VARIANT_MERGE_STRATEGIES = [
+    _NONE_STRING,
+    _MOVE_TO_CALLS_STRING,
+]
```

Comment:
What would be the best way to avoid duplication of these globals between our options and the main pipeline?

Reply:
I think we can move them as public constants inside the MergeOption class....get_variant_merge_strategy can then use those constants. WDYT?

Reply:
Sounds good to me. Using the constants in the pipeline is a bit verbose right now though (e.g. `variant_transform_options.MergeOptions.MERGE_WITH_NON_VARIANTS_STRING`). Is there a good way around this?